ui-tests/src/test/resources/features/integrations/slack.feature

```diff
@@ -160,23 +161,26 @@ Feature: Slack Connector
     And checks that query "select * from contact where company = 'message4' AND first_name = 'syndesis-bot'" has "1" output
 
     # test delay
-    Then send message "messageDelayed" on channel "random"
-    And checks that query "select * from contact where company = 'messageDelayed'" has no output
-    And sleep for "60000" ms
+    When send message "messageDelayed" on channel "slack_connector_test"
+    # test if the message is not arrive immediately
+    And sleep for 2 seconds
+    Then checks that query "select * from contact where company = 'messageDelayed'" has no output
+    When sleep for 60 seconds
     Then checks that query "select * from contact where company = 'messageDelayed' AND first_name = 'syndesis-bot'" has "1" output
```

Comment:
Couldn't the ``waitTillQueryHasOutput`` be used here? Just asking - it seem it would be more dynamic.

Reply:
Yes, It could be. I used that as a solution how to be sure that I am in the first time frame when I am checking the messages which are not there yet. But of course, it can be used generally.